common/pulp/common/plugins/progress.py

```diff
@@ -33,7 +21,7 @@ class ProgressReport(object):
     # If an error occurs outside in progress states, this general failed state can be set
     STATE_FAILED = 'failed'
     # When the user has cancelled a sync
-    STATE_CANCELED = 'cancelled'
+    STATE_CANCELED = 'canceled'
```

Comment:
I think this change might not be safe. The progress reports are part of our API, and I think this might change the progress report API. I recommend leaving this as-is.